server/utils.go

```diff
@@ -0,0 +1,19 @@
+package server
+
+import (
+	"encoding/json"
+
+	"github.com/ava-labs/avalanche-network-runner/rpcpb"
+)
+
+func deepCopy(i *rpcpb.ClusterInfo) (*rpcpb.ClusterInfo, error) {
```

Comment:
Testing this now in HyperSDK: https://github.com/ava-labs/hypersdk/pull/95

Reply:
It may only be necessary to clone the strings in the `chains` field in `CreateSpecificBlockchainsResponse`.

Still researching why any of this would be related (assuming this actually solves the issue)

Reply:
It is possible that we may just need to escape \n properly?

Reply:
It is also possible that there is a race such that `clusterInfo` changes while it is being encoded?